source/includes/_account.md

```diff
@@ -4557,7 +4541,7 @@ accountApi = killbill.api.AccountApi()
 
 account_id = 'c62d5f6d-0b57-444d-bf9b-dd23e781fbda'
 
-auditLogsWithHistory = accountApi.get_account_audit_logs_with_history(account_id, api_key, api_secret)
+accountApi.get_account_audit_logs_with_history(account_id)
```

Comment:
Since this is a retrieve operation, assign the result to a variable like `auditLogsWithHistory`